include/rtcdcpp/PeerConnection.hpp

```diff
@@ -30,8 +30,9 @@
 #include "ChunkQueue.hpp"
 #include "DataChannel.hpp"
 #include "RTCCertificate.hpp"
+#include "Logging.hpp"
 
-#include <log4cxx/logger.h>
+#include <map>
```

Comment:
Actually std::map is used [L141](https://github.com/chadnickbok/librtcdcpp/blob/5c28f4c98d834ab8ee82aea5aeedbf0fdd9bb06b/include/rtcdcpp/PeerConnection.hpp#L141).

<map> seemed to have been implicitly included by log4cxx...